Content.Server/NodeContainer/EntitySystems/NodeContainerSystem.cs

```diff
@@ -46,6 +49,77 @@ public override void Initialize()
             return false;
         }
 
+        public bool TryGetNode<T>(Entity<NodeContainerComponent?> ent, string identifier, [NotNullWhen(true)] out T? node) where T : Node
+        {
+            if (_query.Resolve(ent, ref ent.Comp, false)
+                && ent.Comp.Nodes.TryGetValue(identifier, out var n)
+                && n is T t)
+            {
+                node = t;
+                return true;
+            }
+
+            node = null;
+            return false;
+        }
+
+        public bool TryGetNodes<T1, T2>(
+            Entity<NodeContainerComponent?> ent,
+            string id1,
+            string id2,
+            [NotNullWhen(true)] out T1? node1,
+            [NotNullWhen(true)] out T2? node2)
```

Comment:
I'm not sure whether its better to do this, or to use `..., string id1, out T1? node1, string id2, out T2? node2, ...`

Reply:
I think I normally see this ordering out in the wild but I'm indifferent.